client/src/app/containers/Layout/index.jsx

```diff
@@ -109,4 +98,15 @@ Layout.propTypes = {
   children: PropTypes.node.isRequired,
 };
 
+const useMouseIn = () => {
+  const [mouseIn, setMouseIn] = useState(false);
+  const onMouseEnter = () => {
+    setMouseIn(true);
+  };
+  const onMouseLeave = () => {
+    setMouseIn(false);
+  };
+  return [mouseIn, { onMouseEnter, onMouseLeave }];
+};
```

Comment:
❓  Would it be simpler/possible to use `onHover` instead of this custom logic?

Reply:
Unless you know something I don't, I don't think `onHover` exists... Plus it wouldn't detect leaving the component.